src/Microsoft.Health.Dicom.Api/Features/Filters/PopulateDataPartitionFilterAttribute.cs

```diff
@@ -30,11 +30,8 @@ public sealed class PopulateDataPartitionFilterAttribute : ActionFilterAttribute
     private readonly HashSet<string> _partitionCreationSupportedRouteNames = new HashSet<string>
     {
         KnownRouteNames.PartitionStoreInstance,
-        KnownRouteNames.VersionedPartitionStoreInstance,
         KnownRouteNames.PartitionStoreInstancesInStudy,
```

Comment:
Shouldn't we keep the versioned one and remove the non-versioned route?

Reply:
These are just the route names. They shouldn't have an impact on the actual route (please correct me if I am wrong). Given that now all routes are supposed to be versioned, I didn't like the idea of calling every one of them Versioned.

Reply:
As long as it is same in the controller it should work fine. If you want to remove version, do you want to rename the VersionedRoute to Route and VersionedPartitionedRoute to PartitionedRoute

Reply:
Here's my suggestion - we can use the convention approach to create a convention that will add the API version template at the beginning of all routes at the controller level. Then we can simply convert all `VersionedRoute` attributes to `Route`, and `VersionedPartitionedRoute` to `PartitionedRoute`.

This blog post shows how to do it: https://tpodolak.com/blog/2017/05/04/asp-net-core-api-versioning-convention, although they also do implicit API versioning which I don't think we want to do.

Reply:
I have updated the code. Now we are using conventions.